blackbox/docs/release_notes/1.1.2.txt

```diff
@@ -11,18 +11,23 @@ Released on 2017/04/10.
    If you are upgrading a cluster, you must be running CrateDB 0.57.0 or higher
    before you upgrade to 1.1.2.
 
-   If you want to perform a :ref:`cluster_upgrade`, your current CrateDB version
-   number must be :ref:`version_1.1.1` or higher. Any upgrade from a version
-   prior to this will require a full cluster restart.
+   If you want to perform a :ref:`rolling_upgrade`, your current CrateDB
+   version number must be :ref:`version_1.1.0`. If you want to upgrade from a
```

Comment:
`1.1.1` version `1.1.0` was broken and not available in our distribution channels